Common/TableProducer/multiplicityTable.cxx

```diff
@@ -126,14 +136,21 @@ struct MultiplicityTableTaskIndexed {
     int multTPC = tracksGrouped.size();
     int multNContribs = pvContribsGrouped.size();
 
-    TList* lCalibObjects = ccdb->getForTimeStamp<TList>("Users/v/victor/Centrality/Calibration", 1635634560883); //temporary
-
-    TProfile* hVtxZFV0A = (TProfile*)lCalibObjects->FindObject("hVtxZFV0A");
-    TProfile* hVtxZFT0A = (TProfile*)lCalibObjects->FindObject("hVtxZFT0A");
-    TProfile* hVtxZFT0C = (TProfile*)lCalibObjects->FindObject("hVtxZFT0C");
-    TProfile* hVtxZFDDA = (TProfile*)lCalibObjects->FindObject("hVtxZFDDA");
-    TProfile* hVtxZFDDC = (TProfile*)lCalibObjects->FindObject("hVtxZFDDC");
-    TProfile* hVtxZNTracks = (TProfile*)lCalibObjects->FindObject("hVtxZNTracks");
+    /* check the previous run number */
+    auto bc = collision.bc_as<aod::BCs>();
+    if (bc.runNumber() != mRunNumber) {
+      lCalibObjects = ccdb->getForTimeStamp<TList>("Users/v/victor/Centrality/Calibration", 1635634560883); //temporary
+      if (lCalibObjects) {
```

Comment:
Strictly you wont need this check, if the object is not found the workflow will crash
But it does not harm